app/src/main/res/values/strings.xml

```diff
@@ -448,7 +448,9 @@
     <!-- Title for the customize home screen section with Pocket. -->
     <string name="customize_toggle_pocket_2">Thought-provoking stories</string>
     <!-- Summary for the customize home screen section with Pocket. -->
-    <string name="customize_toggle_pocket_summary">Articles powered by Pocket</string>
+    <string name="customize_toggle_pocket_summary" moz:removedIn="108" tools:ignore="UnusedResources">Articles powered by Pocket</string>
+    <!-- Summary for the customize home screen section with Pocket. The first parameter is product name Pocket -->
+    <string name="customize_toggle_pocket_summary_2">Articles powered by %s</string>
```

Comment:
We could change the original `customize_toggle_pocket_summary` since  that hasn't been exposed to Pontoon yet.